Documentation/api-reference.md

```diff
@@ -117,3 +157,33 @@ consents.put("consents", collectConsents);
 
 Consent.update(consents);
 ```
+
+#### Kotlin
+
+##### Syntax
+```kotlin
+fun update(consents: Map<String, Object>)
+```
+
+##### Example
+```kotlin
+// example 1, updating users collect consent to 'yes'
+val collectConsents = mutableMapOf<String, Any>()
+collectConsents["val"] = "y"
+
+val consents = mutableMapOf<String, Any>()
+consents["consents"] = collectConsents
```

Comment:
you are missing the "collect" node, should be: consents.collect.val = y / n 
(same for the example below)